gdp-src-build/conf/templates/silk.bblayers.conf

```diff
@@ -0,0 +1,7 @@
+include templates/bblayers.inc
+
+# porter specific layer configuration
+BBLAYERS += " \
+  ${TOPDIR}/../meta-renesas \
+  ${TOPDIR}/../meta-renesas/meta-rcar-gen2 \
```

Comment:
I presume this is stemming from the same gdp image bb append, from the include: https://github.com/slawr/meta-renesas/blob/genivi-9-bsp-1.10.0-weston-1.9.0/meta-rcar-gen2/recipes-graphics/images/genivi-demo-platform.bbappend

As I mentioned before my build didn't have this issue, although I think I know why. I may have checked out that sha of meta-renesas, and then re-ran init.sh at somepoint, which will have set it back to the previous commit